stdlib/2and3/codecs.pyi

```diff
@@ -13,10 +13,13 @@ import types
 _Decoded = Text
 _Encoded = bytes
 
+class _Encoder:
+    def __call__(self, input: _Decoded, errors: str = ...) -> Tuple[_Encoded, int]: ...  # signature of Codec().encode
+class _Decoder:
+    def __call__(self, input: _Encoded, errors: str = ...) -> Tuple[_Decoded, int]: ...  # signature of Codec().decode
+
 # TODO: It is not possible to specify these signatures correctly, because
```

Comment:
Also, sounds like we did this TODO :)

Reply:
Removed the TODO now.

Reply:
Actually, maybe that's not quite right—the other Callable aliases down here also take an optional `errors` parameter.

Reply:
I have added a TODO which now says that the others should also be converted to protocol classes.